### ISSUE TITLE
Desktop, Dock and drawer columns to back to their default value

### ISSUE THREAD

**Issue by matttbe** [otakuhqz/Omega#41]
Hello

Thank you for developing this nice app!

I am using the latest version (0.6.3 build 113) and I managed to modify the columns of the desktop, drawer and dock to 7 instead of 5. It was working fine yesterday evening but this morning, it went back to 5.

Now if I change the settings, nothing is modified.

Here are some logs when I try to switch the dock icons from 5 to 7:

```
01-27 07:47:31.002 I/ActivityTaskManager(680): START u0 {act=android.intent.action.MAIN cat=[android.intent.category.HOME] flg=0x10000100 cmp=com.saggitt.omega/.OmegaLauncher (has extras)} from uid 0
01-27 07:47:31.061 W/ActivityManager(680): Unable to start service Intent { act=com.android.launcher3.WINDOW_OVERLAY dat=app://com.saggitt.omega:10278?v=7&cv=9 pkg=com.google.android.googlequicksearchbox } U=0: not found
01-27 07:47:31.120 D/ArtManagerInternalImpl(680): /data/misc/iorapd/com.saggitt.omega/113/com.saggitt.omega.OmegaLauncher/compiled_traces/compiled_trace.pb doesn't exist
01-27 07:47:36.477 I/ActivityTaskManager(680): START u0 {act=android.intent.action.APPLICATION_PREFERENCES flg=0x10000000 pkg=com.saggitt.omega cmp=com.saggitt.omega/.settings.SettingsActivity} from uid 10278
01-27 07:47:36.701 I/ActivityTaskManager(680): Displayed com.saggitt.omega/.settings.SettingsActivity: +224ms
01-27 07:47:39.324 I/ActivityTaskManager(680): START u0 {cmp=com.saggitt.omega/.settings.SettingsActivity (has extras)} from uid 10278
01-27 07:47:39.550 I/ActivityTaskManager(680): Displayed com.saggitt.omega/.settings.SettingsActivity: +225ms
01-27 07:47:40.937 I/JobServiceContext(680): Client timed out while executing (no jobFinished received), sending onStop: 2d29e29 #u0a278/2 com.saggitt.omega/com.android.launcher3.uioverrides.WallpaperManagerCompat$ColorExtractionService
01-27 07:47:48.284 D/InputDispatcher(680): Waiting to send key to Window{2b80a60 u0 com.saggitt.omega/com.saggitt.omega.settings.SettingsActivity} because there are unprocessed events that may cause focus to change
01-27 07:47:48.526 D/InputDispatcher(680): Waiting to send key to Window{da9f2f5 u0 com.saggitt.omega/com.saggitt.omega.settings.SettingsActivity} because there are unprocessed events that may cause focus to change
01-27 07:47:49.450 D/InputDispatcher(680): Waiting to send key to Window{da9f2f5 u0 com.saggitt.omega/com.saggitt.omega.settings.SettingsActivity} because there are unprocessed events that may cause focus to change
01-27 07:47:49.585 W/ActivityManager(680): Unable to start service Intent { act=com.android.launcher3.WINDOW_OVERLAY dat=app://com.saggitt.omega:10278?v=7&cv=9 pkg=com.google.android.googlequicksearchbox } U=0: not found
01-27 07:47:49.711 D/InputDispatcher(680): Waiting to send key to Window{7b29228 u0 com.saggitt.omega/com.saggitt.omega.OmegaLauncher} because there are unprocessed events that may cause focus to change
01-27 07:47:52.518 D/InputDispatcher(680): Waiting to send key to Window{7b29228 u0 com.saggitt.omega/com.saggitt.omega.OmegaLauncher} because there are unprocessed events that may cause focus to change
```

Please tell me if I need to provide something else 🙂
Thanks!

**Comment by matttbe** [comment]
Thanks to the warning in the logs, I just noticed that if I put the search bar back, I can change icons number to 7 again!

Note that if I disable the search bar in the drawer, I cannot use the one of the dock :)

**Comment by bushrang3r** [comment]
V0.7.0
If I go to 
Phone settings > Accessibility > System navigation 
and **change the settings** (gesture navigation, 2 button navigation, or 3 button navigation)
then the number of icons on the homescreen gets reset to the default 5x5.

**Comment by otakuhqz** [comment]
fixed on v0.8
